app/bundles/PluginBundle/Controller/PluginController.php

```diff
@@ -608,4 +608,12 @@ public function reloadAction()
             ]
         );
     }
+
+    public function viewSettingsAction($name)
+    {
+        $integrationHelper = $this->factory->getHelper('integration');
```

Comment:
MauticFactory is deprecated and shouldn't be used. Use `$this->get('mautic.helper.integration')` instead.